doc/source/deployment/setup-deployer.rst

```diff
@@ -87,7 +87,20 @@ There are several ways to install the SUSE Containerized OpenStack software.
    b. sudo zypper install socok8s (installs to /usr/share/socok8s)
```

Comment:
@gracetai5 could you post the steps how to add the iso to the repo list etc. before the zipper install?

Reply:
Is it going to be same as adding repo ?

`zypper ar <iso-image-path> socok8s_iso` 

`zypper ar openSUSE-Addon-socok8s-x86_64-Media.iso socok8s_iso`

Reply:
I copied it from yog's note, cmd similar like this:
sudo zypper addrepo iso:/?iso=/home/ardana/openSUSE-Addon-socok8s-x86_64-Build7.2-Media.iso socok8s

Reply:
I copied it from yog's note, cmd similar like this:
sudo zypper addrepo iso:/?iso=/home/ardana/openSUSE-Addon-socok8s-x86_64-Build7.2-Media.iso socok8s